common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -66,6 +66,23 @@
     @Supported
     public static final KnownKey zimbra_tmp_directory = KnownKey.newKey("${zimbra_home}/data/tmp");
 
+    /**
+     * Sets whether the External Email Warning (EEW) feature is enabled.
+     */
+    @Supported
+    public static final KnownKey zimbra_eew_is_enabled = KnownKey.newKey(false);
```

Comment:
attribute doesn't give enough info about its usage, can change it to like `zimbra_external_email_warning_enabled`

Reply:
Please change the attributes :
zimbra_eew_is_enabled to `zimbra_external_email_warning_enabled`
zimbra_eew_base_admonition to `zimbra_external_email_warning_message`

Reply:
Done.

As an improvement, I've also updated `ZsMsg.properties` so that appropriate help is given to the system administrator when running `zmlocalconfig -i`